apps/src/netsim/NetSimStatusPanel.html.ejs

```diff
@@ -2,10 +2,6 @@
 var i18n = require('./locale');
 %>
 <div class="content-wrap">
-  <% if (remoteNodeName) { %>
-  <p>Connected to <%= remoteNodeName %></p>
-  <% } %>
-
```

Comment:
The status panel body is always hidden right now, but I keep it around to hold diagnostic information (it's easy to show at runtime by removing the 'hidden' class).  This bit of diagnostic would be redundant now though, so I'm tearing it out.